apollo-portal/src/test/java/com/ctrip/framework/apollo/openapi/service/ConsumerServiceIntegrationTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.openapi.service;
+
+import static org.junit.Assert.assertEquals;
+
+import com.ctrip.framework.apollo.portal.AbstractIntegrationTest;
+import com.google.common.collect.Sets;
+import java.util.Set;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.jdbc.Sql;
+
+/**
+ * @author wxq
+ */
+public class ConsumerServiceIntegrationTest extends AbstractIntegrationTest {
+
+  @Autowired
+  private ConsumerService consumerService;
+
+  @Test
+  @Sql(scripts = "ConsumerServiceIntegrationTest.testFindAppIdsAuthorizedByConsumerId.sql", executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
```

Comment:
I dump the part of data from portaldb. Is it a good way to prepare data for test?

Reply:
The data looks ok, however I have some suggestions:
1. The delete statements are not necessary since they will be cleared via `/sql/cleanup.sql`
2. To test the logic, it's better to add more apps, e.g. consumer-test-app-id-0, consumer-test-app-id-1 and consumer-test-app-id-2. And we setup the data that the consumer only has permission to consumer-test-app-id-0 and consumer-test-app-id-1.
3. For integration test, I would suggest we test against the open api `/apps/authorized`, e.g using a rest template to request the path
4. Then the AppControllerTest could be simplified as a pure unit test, which doesn't need to start a server